src/components/molecules/Steps/Steps.scss

```diff
@@ -0,0 +1,90 @@
+@import '../../../assets/styles/breakpoints';
+@import '../../../assets/styles/variables';
+
+.steps {
```

Comment:
@Rajmohanrk We are using branded classnames to avoid future conflicts. So for .steps you could use `.cg-steps`